packages/webostv/template/package.json

```diff
@@ -33,13 +33,13 @@
     "dist/*"
   ],
   "dependencies": {
-    "@enact/core": "^3.2.0",
-    "@enact/i18n": "^3.2.0",
+    "@enact/core": "^3.3.0-alpha.11",
+    "@enact/i18n": "^3.3.0-alpha.11",
     "@enact/sandstone": "enyojs/sandstone",
-    "@enact/spotlight": "^3.2.0",
-    "@enact/ui": "^3.2.0",
-    "@enact/webos": "^3.2.0",
-    "ilib": "npm:ilib-webos-tv@^14.4.0-webostv1",
+    "@enact/spotlight": "^3.3.0-alpha.11",
+    "@enact/ui": "^3.3.0-alpha.11",
+    "@enact/webos": "^3.3.0-alpha.11",
```

Comment:
Would it make sense to update the enact deps to `3.3.1` at this point, now that we've officially released it?